README.md

```diff
@@ -23,155 +23,80 @@
 
 The Pulsar Node.js client can be used to create Pulsar producers and consumers in Node.js.
 
-## Requirements
-
-Pulsar Node.js client library is based on the C++ client library. Follow the instructions for
-[C++ library](https://pulsar.apache.org/docs/en/client-libraries-cpp/) for installing the binaries through
-[RPM](https://pulsar.apache.org/docs/en/client-libraries-cpp/#rpm),
-[Deb](https://pulsar.apache.org/docs/en/client-libraries-cpp/#deb) or
-[Homebrew packages](https://pulsar.apache.org/docs/en/client-libraries-cpp/#macos).
-
-(Note: you will need to install not only the pulsar-client library but also the pulsar-client-dev library)
-
-Also, this library works only in Node.js 10.x or later because it uses the
+This library works only in Node.js 10.x or later because it uses the
 [node-addon-api](https://github.com/nodejs/node-addon-api) module to wrap the C++ library.
 
-## Compatibility
-
-Compatibility between each version of the Node.js client and the C++ client is as follows:
-
-| Node.js client | C++ client      |
-|----------------|-----------------|
-| 1.0.x          | 2.3.0 or later  |
-| 1.1.x          | 2.4.0 or later  |
-| 1.2.x          | 2.5.0 or later  |
-| 1.3.x          | 2.7.0 or later  |
-| 1.4.x - 1.6.x  | 2.8.0 or later  |
-| 1.7.x          | 2.10.1 or later |
-
-If an incompatible version of the C++ client is installed, you may fail to build or run this library.
-
 ## How to install
 
-### Install on windows
+> **Note**
+>
+> Only available for versions after 1.8.0. For versions before 1.8.0, you need to install the C++ client first, and switch to the corresponding version branch to view the specific steps.
 
-1. Build the Pulsar C++ client on windows.
+1. You can use `npm` or `yarn` to install `pulsar-client-node`
```

Comment:
Now we don't need to use multiple steps. I suggest keeping only the install instructions:

````diff
diff --git a/README.md.old b/README.md
index 2f781c9..fbb4d2f 100644
--- a/README.md.old
+++ b/README.md
@@ -32,18 +32,19 @@ This library works only in Node.js 10.x or later because it 
uses the
 >
 > Only available for versions after 1.8.0. For versions before 1.8.0, you need 
to install the C++ client first, and switch to the corresponding version branch 
to view the specific steps.
 
-1. You can use `npm` or `yarn` to install `pulsar-client-node`
+### `npm`
 
 ```shell
 npm install pulsar-client
-#or
-yarn add pulsar-client
 ```
 
-2. Run Sample code
+### `yarn`
 
-Please refer to [examples](https://github.com/apache/pulsar-client-node/tree/master/examples).
+```shell
+yarn add pulsar-client
+```
 
+After install, you can run the [examples](https://github.com/apache/pulsar-client-node/tree/master/examples).
 
 ### Prebuilt binaries
````

Reply:
oh, Let us only keep `npm`.